core/src/main/java/greencity/controller/OrderController.java

```diff
@@ -530,4 +531,25 @@ public ResponseEntity<List<LocationsDto>> getAllLocationsByCourierId(
         List<LocationsDto> locations = ubsClientService.getAllLocationsByCourierId(courierId);
         return ResponseEntity.status(HttpStatus.OK).body(locations);
     }
+
+    /**
+     * Receives and processes payment information from the Monobank API. This method
+     * handles the incoming payment response and validates the payment details
+     * provided by Monobank. The payment details are logged and then passed to the
+     * {@code validatePaymentFromMonoBank} method for further validation and
+     * processing.
+     *
+     * @param response the payment response received from Monobank, containing
+     *                 details such as transaction ID, status, and amount.
+     */
+    @Operation(summary = "Receive payment information from Monobank API")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = HttpStatuses.OK),
+        @ApiResponse(responseCode = "400", description = HttpStatuses.BAD_REQUEST)
+    })
+    @PostMapping("/monobank/payments")
+    public void receivePaymentFromMonoBank(@RequestBody MonoBankPaymentResponseDto response) {
+        log.info("Response from MONOBANK API: {}", response);
```

Comment:
do you need to log the whole request?
Please check if it does not contain sensitive info

Reply:
I did not have the opportunity to test this case locally, so I added temporary logging to verify its correctness. If it works correctly, I will remove the logging.